lib/aliquot-pay.rb

```diff
@@ -181,23 +216,21 @@ def build_signature
             ensure_intermediate_key
           end
 
-    signature_string =
-      signed_string_message = ['Google',
-                               recipient_id,
-                               @protocol_version.to_s,
-                               signed_message_string].map do |str|
-        [str.length].pack('V') + str
-      end.join
+    signature_string = ['Google',
+                        recipient_id,
+                        @protocol_version.to_s,
+                        signed_message_string].map do |str|
+      [str.length].pack('V') + str
+    end.join
```

Comment:
Well, that the `do` and `end` don't somehow adhere to shared indentation.

Reply:
This ways auto-indented with rubocop. I agree it looks annoying. The only change I did to this line though was to remove `signed_string_message` since it seemed superfluous maybe I should revert such changes such that we don't use too much time discussing this.

Reply:
Completely fair :+1: Feel free to resolve.